runner/run.go

```diff
@@ -130,6 +130,34 @@ func Run(ctx context.Context, wg *sync.WaitGroup, globalCollectionOpts state.Col
 		return
 	}
 
+	if globalCollectionOpts.GenerateExplainAnalyzeHelperSql != "" {
+		wg.Add(1)
+		testRunSuccess = make(chan bool)
```

Comment:
It doesn't have to be part of this PR, but we may want to add test run output for the presence of the helper function and success on running a test explain.

Reply:
Yeah, that's a good point. I'll add this in a follow-up PR.